code/modules/mob/living/carbon/carbon.dm

```diff
@@ -883,6 +883,8 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 				legcuffed = null
 				toggle_move_intent()
 				update_inv_legcuffed()
+				if(istype(I, /obj/item/restraints/legcuffs/bola/energy))
```

Comment:
Not a super fan of this snowflaking, you may want to consider moving the `qdel` in the energy bola's `dropped` proc (only when it has hit someone)

Reply:
Will look into it

Reply:
hopefully this is better?